test/azure/generated/AzureCompositeModelClient/operations/primitive.ts

```diff
@@ -2048,8 +2047,8 @@ export class Primitive {
     let field = (options && options.field !== undefined) ? options.field : undefined;
     // Validate
     try {
-      if(field && !moment.isDuration(field)) {
-        throw new Error('field must be of type moment.duration.');
+      if(field && !msRest.isDuration(field)) {
+        throw new Error('field must be of type string.');
```

Comment:
This is what happens when you generalize... There's a template string in the generator `"{0} must be of type {1}"` so we might as well just live with the lousy message here and wait for the new generator to improve it.